barretenberg/cpp/docs/src/sumcheck-outline.md

```diff
@@ -0,0 +1,5 @@
+# Sumcheck Implementation
+
+Let \f$ P_1, \ldots, P_N \f$ be multilinear polynomials in \f$ d \f$ variables. We also refer them as Honk polynomials. 
```

Comment:
```suggestion
Let \f$ P_1, \ldots, P_N \f$ be multilinear polynomials in \f$ d \f$ variables. We also refer to them as Honk polynomials. 
```

Reply:
Oh, that was a template that I abandoned. Now I am not sure what to include there to not copy-paste SumcheckProver class description. 
Should I just rewrite it and update PR tomorrow?